app/controllers/dashboard_controller.rb

```diff
@@ -1,7 +1,10 @@
 class DashboardController < ApplicationController
   before_action :require_default
-  
-  def show
 
+  def show
+    @favorited_companies = current_user.favorite_companies
+    @favorite_members = current_user.favorite_users
+    @companies = current_user.companies
+    @feedback = @companies.questions(current_user.companies)
```

Comment:
might want to use a presenter here?

Reply:
That's a great idea. I'll keep it as a refactor.